src/QMCDrivers/DriverTraits.h

```diff
@@ -18,7 +18,8 @@ enum class QMCRunType
   WF_TEST,
   VMC_BATCH,
   DMC_BATCH,
-  OPTIMIZE_BATCH
+  OPTIMIZE_BATCH,
```

Comment:
Is OPTIMIZE_BATCH actually used?

Reply:
I mean we use linear not opt. The same as we should only use linear_batch and there is no need to expose opt_batch even we need the class.

Reply:
For now, it seems necessary to distinguish between the batched and non-batched versions.   After the non-batched versions get removed, this enum could be simplified.    But I'm not sure if I understood your comment correctly?